webui/react/src/pages/ExperimentList.tsx

```diff
@@ -308,7 +315,49 @@ const ExperimentList: React.FC = () => {
       value ? <Link path={paths.experimentDetails(value)}>{value}</Link> : null
     );
 
-    const tableColumns: ColumnsType<ExperimentItem> = [
+    const tableColumns: ExperimentColumnDefs = {
+      action:
+      {
+        align: 'right',
+        className: 'fullCell',
+        fixed: 'right',
+        key: 'action',
+        onCell: () => ({ isCellRightClickable: true } as React.HTMLAttributes<HTMLElement>),
+        render: actionRenderer,
+        title: '',
+        width: DEFAULT_COLUMN_WIDTHS['action' as ExperimentColumnName],
+      },
+      archived:
```

Comment:
chore: I'm not sure what's causing this, but toggling the archived slider isn't working on Netlify. Once the archived columns appears, it won't disappear.

Reply:
Archived also shouldn't be an option in the column picker modal. This is probably related.